Resources/AzureDevOps/StepTemplates/deploy-arm-template.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  AzureSubscription: 'SFA-CDH-Dev/Test (962cae10-2950-412a-93e3-d8ae92b17896)'  
+  SetSubscriptionScript: 'https://raw.githubusercontent.com/SkillsFundingAgency/dfc-devops/master/PSScripts/Set-EsfaResourceGroupTags.ps1'
+  EnvironmentTag: 'DEV/Test'
```

Comment:
default should be ''

Reply:
See previous comment about the Environment, ParentBusiness & ParentBusiness tags used for the resource group creation. Blame the silly convention on Core team :)